src/operator-docs/filtering/filter.ts

```diff
@@ -1,6 +1,79 @@
 import { OperatorDoc } from '../operator.model';
 
 export const filter: OperatorDoc = {
-  'name': 'filter',
-  'operatorType': 'filtering'
+  name: 'filter',
+  operatorType: 'filtering',
+  signature:
+    'public filter(predicate: function(value: T, index: number): boolean, thisArg: any): Observable',
+  parameters: [
+    {
+      name: 'predicate',
+      type: 'function(value: T, index: number): boolean',
+      attribute: '',
+      description: `A function that evaluates each value emitted by the source Observable.
+         If it returns true, the value is emitted, if false the value is not passed to the output Observable.
+        The index parameter is the number i for the i-th source emission that has happened since the subscription,
+         starting from the number 0.`
+    },
+    {
+      name: 'thisArg',
+      type: 'any',
+      attribute: 'optional',
+      description:
+        'An optional argument to determine the value of this in the predicate function.'
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/filter.png',
+  shortDescription: {
+    description:
+      'Filter items emitted by the source Observable by only emitting those that satisfy a specified predicate.',
+    extras: [
+      {
+        type: 'Tip',
+        text: `
+        Like
+        <a
+         target="_blank"
+         href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter">
```

Comment:
Love this! Adding these extra links to key terms is really valuable in my opinion. 👍